### PR TITLE
Get photo of user 

### PR DESCRIPTION
**Pull request by olegatr**
Get photo  of user by sugar syntax.
For example see implementation message 'show_my_photo' in /examples/bot.rb

### CODE REVIEW

**Comment by ivanovaleksey** [comment]
@olegatr you have very strange commit name

**Comment by olegatr** [comment]
Yep.. very strange :) I hurried to get a child from kindergarten.

File: lib/telegram/bot/types/user.rb [hunk 1]
```diff
@@ -6,6 +6,16 @@ class User < Base
         attribute :first_name, String
         attribute :last_name, String
         attribute :username, String
+
+        attr_accessor :id
```
Comment (ivanovaleksey) [inline lib/telegram/bot/types/user.rb]: Why is it here?
`attribute :id` already provides `attr_accessor` for `id`

File: lib/telegram/bot/types/user.rb [hunk 2]
```diff
@@ -6,6 +6,16 @@ class User < Base
...
+
+        def profile_photos(api, offset = nil, limit = nil)
+          response = api.getUserProfilePhotos(user_id: id,
+                                              offset: offset, limit: limit)
+          response['result']['photos'] = response['result']['photos'].flatten
```
Comment (ivanovaleksey) [inline lib/telegram/bot/types/user.rb]: why not `flatten!`?

File: lib/telegram/bot/types/user.rb [hunk 3]
```diff
@@ -6,6 +6,16 @@ class User < Base
...
+          return nil unless response['ok']
```
Comment (ivanovaleksey) [inline lib/telegram/bot/types/user.rb]: just `return unless response['ok']`

File: lib/telegram/bot/types/photo_size.rb [hunk 4]
```diff
@@ -6,6 +6,16 @@ class PhotoSize < Base
         attribute :width, Integer
         attribute :height, Integer
         attribute :file_size, Integer
+
+        attr_accessor :file_id
+
+        def get_file(api)
```
Comment (ivanovaleksey) [inline lib/telegram/bot/types/photo_size.rb]: As for me it looks very dirty to pass `Api` instance as parameter.
I can't provide better solution now, just my thoughts.

File: lib/telegram/bot/types/user.rb (hunk 1 above)
Comment (ivanovaleksey) [inline lib/telegram/bot/types/user.rb]: `attribute :id` does define attr_accessor for id, doesn't it?

**Comment by ivanovaleksey** [comment]
@olegatr this repo doesn't follow strictly to any commit naming conventions but it's worth to read http://tbaggery.com/2008/04/19/a-note-about-git-commit-messages.html
Don't forget to squash commits in the end.

File: lib/telegram/bot/types/user.rb (hunk 1 above)
Comment (olegatr) [inline lib/telegram/bot/types/user.rb]: you right

File: lib/telegram/bot/types/photo_size.rb (hunk 4 above)
Comment (olegatr) [inline lib/telegram/bot/types/photo_size.rb]: I think so also. Another way, needs to change the design.

File: lib/telegram/bot/types/user.rb (hunk 3 above)
Comment (olegatr) [inline lib/telegram/bot/types/user.rb]: Done

File: lib/telegram/bot/types/user.rb (hunk 2 above)
Comment (olegatr) [inline lib/telegram/bot/types/user.rb]: Done

File: lib/telegram/bot/types/photo_size.rb (hunk 4 above)
Comment (ivanovaleksey) [inline lib/telegram/bot/types/photo_size.rb]: Maybe @atipugin could take a look and give some advise about the whole thing.

**Pull request closed without merging**